src/Symfony/Component/Finder/Tests/GitignoreTest.php

```diff
@@ -113,6 +113,21 @@ public function provider()
                 ['app/cache/file.txt', 'app/cache/subdir/ile.txt'],
                 ['a/app/cache/file.txt'],
             ],
+            [
+                '
+                /app/cache/
+                #LastLineIsComment',
+                ['app/cache/file.txt', 'app/cache/subdir/ile.txt'],
+                ['a/app/cache/file.txt'],
+            ],
+            [
+                '
+                /app/cache/
+                \#EscapedComment
```

Comment:
What  about using a real file name to check that it will be in the list (`#file.txt` like in a test above)?

Reply:
That'd indeed be more useful, I'll change it.